### PR TITLE
Generate search pager link

### PR DESCRIPTION
**Pull request by CarsonF**
Generate search pager link

### CODE REVIEW

**Comment by GawainLynch** [comment]
Why no `static` to replace the hard coding?

**Comment by CarsonF** [comment]
:koala: told me not too

**Comment by GawainLynch** [comment]
Damn :koala: was probably eating those damn :leaves: again!

File: src/Controllers/Frontend.php [hunk 1]
```diff
@@ -367,7 +367,7 @@ public function search(Request $request, Silex\Application $app)
             'current'      => $page,
             'showing_from' => $offset + 1,
             'showing_to'   => $offset + count($result['results']),
-            'link'         => '/search?q=' . rawurlencode($q) . '&page_search='
+            'link'         => $app['url_generator']->generate('search', ['q' => $q]) . '&page_search='
```
Comment (GawainLynch) [inline src/Controllers/Frontend.php]: Just a minor point… and stuff… but 2.2 still supports PHP 5.3.stupid

**Comment by CarsonF** [comment]
I don't think the failure is my fault

**Comment by GawainLynch** [comment]
Unless you're the one that committed breaking changes to the last 4 revisions of Codeception, then no, you're not to blame for those fails.

**Comment by bobdenotter** [comment]
Merging in! :-)

**Pull request merged**